src/components/SearchField.tsx

```diff
@@ -1,38 +1,241 @@
-import React, { useState } from 'react';
+import React, { useCallback, useState } from 'react';
+import { useHistory } from 'react-router';
+
+import '../styles/SearchField.scss';
 
 const loupeIcon = '/images/search-icon.svg';
 
+const genres = [
+  'none',
+  'Comedy',
```

Comment:
It seems like it could be enum

Reply:
i dont know how to make my code working with enum, can you help me this please?

Reply:
Could you explain a bit what is not working?

Reply:
enum Genres {
  Drama='Drama',
...
}
use like this: Genres.Drama

Reply:
@Kachyr any update on this?

Reply:
Done thank you)